### PR TITLE
Apply font based on service: SectionLabel

### PR DESCRIPTION
**Pull request by ghost**
Contributes to #543 

**Overall change:** Apply font style based on the `service`
**Code changes:**

- Add a required `service` prop on internal `Title` and `SectionLabel`
- Update tests and stories
- Update `psammead-styles` to `1.1.0` and `storybook-helpers` to `3.1.0`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by j-pendlebury — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]

File: packages/components/psammead-section-label/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.0.0 | [PR#???](https://github.com/bbc/psammead/pull/???) Apply font based on service prop |
```
Comment (andrew-nowak) [inline packages/components/psammead-section-label/CHANGELOG.md]:
```suggestion
| 2.0.0 | [PR#974](https://github.com/bbc/psammead/pull/974) Apply font based on service prop |
```

File: packages/components/psammead-section-label/src/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -491,8 +497,9 @@ exports[`SectionLabel Without bar With title should render correctly with arabic
   line-height: 2rem;
   color: #222222;
   background-color: #FFFFFF;
-  font-family: ReithSans,Helvetica,Arial,sans-serif;
-  font-weight: normal;
+  font-family: NassimPersian,Arial,Verdana,Geneva,Helvetica,sans serif;
```
Comment (andrew-nowak) [inline packages/components/psammead-section-label/src/__snapshots__/index.test.jsx.snap:500]: I know it's not part of this PR but shouldn't this be `sans-serif`?

Comment (andrew-nowak) [reply packages/components/psammead-section-label/src/__snapshots__/index.test.jsx.snap:500]: Raised #1085 to fix this

**Review by andrew-nowak — APPROVED** [review]
One (hopefully) last merge conflicts to solve, then lgtm

**Comment by david-boydell** [comment]
All good, thanks this is ready for merge.

File: packages/components/psammead-section-label/package.json [hunk 3]
```diff
@@ -22,12 +22,12 @@
   },
   "homepage": "https://github.com/bbc/psammead/blob/latest/packages/components/psammead-section-label/README.md",
   "dependencies": {
-    "@bbc/gel-foundations": "^3.0.3",
-    "@bbc/psammead-styles": "^1.1.3"
+    "@bbc/gel-foundations": "^3.0.2",
```
Comment (amywalkerdev) [inline packages/components/psammead-section-label/package.json]: Using older versions? Surely this isn't correct

**Review by amywalkerdev — CHANGES_REQUESTED** [review]
Older versions of packages being used

**Comment by david-boydell** [comment]
Coolio, all good.

**Review by amywalkerdev — APPROVED** [review]

**Pull request merged**